pglast/__main__.py

```diff
@@ -86,7 +87,9 @@ def main(options=None):
                         help='a file containing the SQL statement to be pretty-printed,'
                         ' by default stdin, when not specified with --statement option')
     parser.add_argument('outfile', nargs='?', type=argparse.FileType('w'),
-                        help='where the result will be written, by default stdout')
+                        help='where the result will be written, by default stdout'),
+    parser.add_argument('-F', '--remove_pg_catalog_from_functions', action='store_true', default=False,
```

Comment:
Please use a dash (``-``) instead of an underscore (``_``) in the long option name, for consistency.
Also, move it before the ``infile`` argument, to keep it together with the other "named" options.